database_scripts/tabels/create_orderItem.sql

```diff
@@ -1,14 +1,15 @@
 use customer_db
 
-create table dbo.OrderItem
-(
-ID int  NOT NULL PRIMARY KEY,
-OrderID int FOREIGN KEY dbo.Order,
-ProductID int FOREIGN KEY dbo.Product,
-UnitPrice decimal(10,2),
-Price int,
-TotalPrice decimal(10,2),
-Quantity int,
-UpdatedDate datetime,
-CreatedDate datetime
-)
\ No newline at end of file
+CREATE TABLE [dbo].[OrderItems] (
+    [ID] [int] NOT NULL IDENTITY,
+    [OrderID] [int],
+    [ProductID] [int],
+    [UnitPrice] [decimal](18, 2),
+    [TotalPrice] [decimal](18, 2),
+    [Quantity] [int],
+    [UpdatedDate] [datetime],
+    [CreatedDate] [datetime],
+    CONSTRAINT [PK_dbo.OrderItems] PRIMARY KEY ([ID])
+)
+CREATE INDEX [IX_OrderID] ON [dbo].[OrderItems]([OrderID])
+CREATE INDEX [IX_ProductID] ON [dbo].[OrderItems]([ProductID])
```

Comment:
Tu jest ok, tylko musisz dodać NOT NULL przy polach, które są IsRequired(), zmień `use customer_db` na `use BProject_db` oraz idexy przenieś do contraintów